layouts/_default/baseof.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="{{ .Site.LanguageCode }}">
+  <head>
+  <meta charset="utf-8">
```

Comment:
Pour une meilleure lisibilité, tu pourrais passer tout le header dans un partial spécifique `{{ partial "head.html" . }}`.

Reply:
✅ done in https://github.com/jamstatic/jamstatic-fr/pull/70/commits/a93ce5d91764816bccee388c1c120ba0d09137d2